externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/schema/RowSet.scala

```diff
@@ -97,12 +98,14 @@ object RowSet {
     val timeFormatters = HiveResult.getTimeFormatters
     var i = 0
     val columnSize = schema.length
+    val tColumns = new java.util.ArrayList[TColumn](columnSize)
```

Comment:
nit: usually we use `util.ArrayList` or `JArrayList` to reference Java collection classes

Reply:
Will try to adopt your advice in the following cleaning-up PRs.